app/assets/stylesheets/entrypoints/component_preview.scss

```diff
@@ -14,4 +14,5 @@ $font-family-sans-serif: "Open Sans", Arial, Helvetica, sans-serif;
 
 body {
   padding: 1rem;
+  background-color: var(--tblr-white);
```

Comment:
set bg white for lookbook because tabler's bg color for input is transparent, 
otherwise will be this
![image](https://github.com/user-attachments/assets/9ad03f86-0275-4c62-ac27-dcad9712ae0f)
 instead of this
![image](https://github.com/user-attachments/assets/e9a0a0b9-ed3c-42d3-ab3e-795276737d8a)